core/rats_init.c

```diff
@@ -0,0 +1,173 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// clang-format off
+#include <stdlib.h>
+#include <string.h>
+#include <librats/log.h>
+#include <librats/err.h>
+#include "librats/attester.h"
+#include <internal/attester.h>
+#include "librats/verifier.h"
+#include <internal/verifier.h>
+#include <internal/core.h>
+
+#ifdef SGX
+#include "rats_t.h"
+#endif
+// clang-format on
+
+#ifdef SGX
+	// clang-format off
+#define RATS_ATTESTER_NUM	5
+#define RATS_VERIFIER_NUM	5
+#define RATS_NAME   		32
+// clang-format on
+char rats_attester_name[RATS_ATTESTER_NUM][RATS_NAME] = { "nullattester", "sgx_ecdsa", "sgx_la" };
+char rats_verifier_name[RATS_VERIFIER_NUM][RATS_NAME] = { "nullverifier", "sgx_ecdsa",
+							  "sgx_ecdsa_qve", "sgx_la" };
+#endif
+
+rats_attester_err_t rats_collect_init(rats_conf_t *conf, rats_core_context_t *ctx)
+{
```

Comment:
May be `rats_attester_init`  is more clairy than `rats_collect_init`, since the following function is called `rats_verify_init `.

Reply:
rats_attester_init has been used by others